auth-server/README.adoc

```diff
@@ -237,6 +237,12 @@ public class ClientApplication {
 }
 ----
 
+NOTE: You ought to create the `ClientApplication` either in a different
+package of the `SocialApplication` class, or in the maven `test`
+folder. Otherwise, `OAuth2SsoDefaultConfiguration` configuration class
```

Comment:
It's unclear what you mean by "the maven 'test' folder". Also unclear where the `OAuth2SsoDefaultConfiguration` is defined. I think all you really need to say is that the `ClientApplication` should not be in the same package or a subpackage of `SocialApplication`?